docs/book/src/developer/tilt.md

```diff
@@ -71,8 +71,31 @@ documentation](https://docs.tilt.dev/api.html#api.default_registry) for more det
 **enable_providers** (Array[]String, default=['docker']): A list of the providers to enable. See [available providers](#available-providers)
 for more details.
 
+**flavor_dirs** (Map{String: Array[]String}, default={"docker": [
+"./test/infrastructure/docker/templates"]}): A map of providers to directories containing flavor templates. An example of the field is given below. See [Deploying a workload cluster](#deploying-a-workload-cluster) for how this is used.
+
+```json
+"flavor_dirs": {
+    "docker": [
+        "./test/infrastructure/docker/templates",
+        "<other-template-dir>"
+    ],
+    "azure": [
+        "<azure-template-dir>"
+    ],
+    "aws": [
+        "<aws-template-dir>"
+    ],
+    "gcp": [
+        "<gcp-template-dir>"
+    ]
+}
```

Comment:
```suggestion
flavor_dirs:
  docker:
  - ./test/infrastructure/docker/templates
  - <other-template-dir
  azure:
  - <azure-template-dir>"
  aws:
  - <aws-template-dir>"
  gcp:
  - <gcp-template-dir>
```
nit: to be consistent with the rest of the doc

Reply:
Should we call it template_dirs? Usually we call this template relatively consistently (I think the only place where we call it flavor is when you select a cluster template on the clusterctl cli)